doc/administrator_guide.md

```diff
@@ -94,6 +94,28 @@ Select the account key for use with Google Cloud API (which kcidb uses):
 Clone kcidb repository, checkout the revision you want to deploy, and install
 kcidb as described in [README.md](README.md).
 
+### Additional Dependencies
+
+While deploying you may also need to install psql. psql is a command-line tool for interacting with the PostgreSQL database. It is used to manage and manipulate data stored in a PostgreSQL database, as well as to administer and configure the database itself. 
```

Comment:
Could you wrap this line to 78 columns?

Also, could you please mention that the instructions below are only meant for Debian-based distros, such as Ubuntu. It would be nice to add instructions for Fedora-based distros as well, as they're also popular, but we could do without them for the start, as long as we make clear that psql is required.